magento/magento2
# Enabling Individual Editing of Shipping and Billing Addresses from the Customer Dashboard

### Description (*)
We propose an enhancement to the Magento customer dashboard's address management functionality. 
Currently, when adding an address, it defaults to both billing and shipping. However, distinguishing between them becomes confusing for clients, as editing one address impacts both. **_This native workflow requires generating a new address to differentiate between default shipping or billing_**. Our enhancement streamlines this process by **_allowing direct individual editing of billing and shipping addresses,_** thereby simplifying the user experience for logged-in clients.

Video 1: "Native Magento Flow: Editing Billing and Shipping Addresses"
[NativeMagento.webm](https://github.com/magento/magento2/assets/115582581/d9851c8a-8966-4a3a-90b1-6cb4315edece)


Video 2: "Improvement Proposal: Individual Editing of Billing and Shipping Addresses"
[addressImprovement.webm](https://github.com/magento/magento2/assets/115582581/30996b4a-0e13-4ca8-bcd3-1c8f5b01eaf2)


### Manual testing scenarios (*)

1. Navigate to the customer dashboard.
2. Add a new address and verify that it defaults to both billing and shipping.
3. Edit the address and observe that both billing and shipping are affected.
4. Apply this pull request's changes and verify that individual editing of billing and shipping addresses is possible without affecting the other.


### Questions or comments



### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#38678: Enabling Individual Editing of Shipping and Billing Addresses from the Customer Dashboard
